From 35c5b1493823b37c5ad8c89ac47cdfeebadb4307 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Sat, 29 Jan 2022 02:08:14 +0200
Subject: [PATCH] [SW-65638] gaudi2: release: explicit files copy

Only Copy the revision files as in update procedure, without the
any additional operation.

Change-Id: I197115fc96472012f6704839884b3a4aeb9e466a
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/scripts/fwi/fw_installer.sh b/scripts/fwi/fw_installer.sh
index c70e4b5..0f2bbfc 100755
--- a/scripts/fwi/fw_installer.sh
+++ b/scripts/fwi/fw_installer.sh
@@ -1438,6 +1438,11 @@
 	return 0
 }
 
+function copy_files_op()
+{
+	[[ $copy_files_f -ne 1 ]] && return 0
+	user_break_stage="copy_files"
+}
 function adjust_pci_oam_mapping()
 {
 	local file=$1
@@ -1887,6 +1892,7 @@
 	mods_var_register rmt_host rmt_user rmt_passwd
 	mods_var_register create_cfg snapshot
 	mods_var_register config_link show_env
+	mods_var_register copy_files
 	# User config files and platform selection was already
 	# handled in the early param handling
 	load_platform_api ${platform_type}
@@ -1914,6 +1920,7 @@
 	api_sanity_check
 	[ $? -ne 0 ] && return 1
 	handle_ignore_op
+	copy_files_op
 	run_stages
 	pinfo "For more details see logs directory ${log_dir}/"
 }
